client/src/actions/index.ts

```diff
@@ -19,6 +19,7 @@ import * as selnActions from "./selection";
 import * as viewActions from "./viewStack";
```

Comment:
`actions/index.ts` now re-exports the annotation thunks, so most components can just import from `actions` without caring about the new module layout.